webview/cocoa.py

```diff
@@ -24,6 +24,42 @@
 info['NSAppTransportSecurity'] = {'NSAllowsArbitraryLoads': Foundation.YES}
 
 
+class DragBar(AppKit.NSView):
+    def mouseDragged_(self, theEvent):
+        screenFrame = AppKit.NSScreen.mainScreen().frame()
+        if screenFrame is None:
+            sys.stderr.write('failed to obtain screen\n')
+            raise RuntimeError
```

Comment:
It would be better to raise a `RuntimeError('Failed to obtain screen')`. Same with the rest of RuntimeErrors